hal/lib/athena/Accelerometer.cpp

```diff
@@ -99,7 +99,7 @@ static void initializeAccelerometer() {
 
 static void writeRegister(Register reg, uint8_t data) {
   int32_t status = 0;
-  uint32_t initialTime;
+  uint64_t initialTime;
```

Comment:
No. GetFPGATime returns a uint64_t, which is what this number is received from. Its one of the few functions still with unsigned ints in the HAL, and that's one I don't think we want to change.